### DISCUSSION TITLE
Make MultiSelect sortable

### DISCUSSION THREAD

**Discussion by baba43** [mantinedev/mantine#3619] [Feature requests]
I use the MultiSelect for entering keywords and would love it if there was a sort function by default (i.e. a way for the user to edit the order of items). I can imagine that this would be interesting for several use cases.

**Comment by ks-himanshub** [comment]
for that you can sort the array first and then inject it in the data props of multiselect component

**Reply by baba43** [reply]
Yes, but I want the user to be able to rearrange the items.

**Reply by bogordesaincom** [reply]
you can use refetch and array map sort again.
